login.html

```diff
@@ -1,39 +1,33 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
   <title>HTML & CSS Training</title>
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
-  <link rel="stylesheet"  type="text/css" href="application.css">
+  <link rel="stylesheet" type="text/css" href="application.css">
 </head>
+
 <body>
-  <div class= "login-container">
-    <div class= "border">
-    </div>
-    <img class= "wolox-logo"
-      src= "https://trello-attachments.s3.amazonaws.com/607457e8ad4162814da41ef0/607457e8ad4162814da41f08/x/ab567d29d6f06a584a4102325415d39f/LogoWolox.png"
-      alt= "wolox logo"
-    >
-    <p class= "text"> Email </p>
-    <div class= "credentials-box">
-    </div>
-    <p class= "text"> Password </p>
-    <div class= "credentials-box">
-    </div>
-    <div class= "button">
-        <a href= "book-list.html" class = "login-button">
-          <p class= "login-text"> Login </p>    
+  <div class="master-container login-master-container">
+    <div class="login-container">
+      <img class="wolox-logo" src="Images/LogoWolox.png" alt="wolox logo">
+      <form class="login-form">
+        <label class="text"> Email </label>
+        <input type="email" class="credentials-box" />
```

Comment:
Just a suggestion for the future: Keep in mind that assigning a "for" param in the label equal to the "id" of the input it's a good practice (more info on why and how [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)). For example:
```suggestion
        <label for="email" class="text"> Email </label>
        <input type="email" class="credentials-box" id="email" />
```